content/tutorials/publish-subscribe.textile

```diff
@@ -293,22 +293,37 @@ blang[android].
   "See this step in Github":https://github.com/ably/tutorials/commit/pubsub-android-step2
 
 blang[javascript].
-  To start using Ably in your web app, you first need to include the Ably library. We recommend that you include the latest client library from our CDN using a simple @<script>@ tag. The client library must be instantiated with the API key you copied in Step 1. Note in production we recommend you always use the "token authentication scheme":/core-features/authentication#token-authentication for browser clients, however in this example we use an API key for simplicity.
+  First, create a blank HTML page called @example.html@, to host your web app:
```

Comment:
I think this can just be "Create", especially as "First, create" is also used later.